core/commands/commands_test.go

```diff
@@ -15,63 +15,6 @@ func collectPaths(prefix string, cmd *cmds.Command, out map[string]struct{}) {
 	}
 }
 
-func TestROCommands(t *testing.T) {
-	list := []string{
-		"/block",
-		"/block/get",
-		"/block/stat",
-		"/cat",
-		"/commands",
-		"/commands/completion",
-		"/commands/completion/bash",
-		"/commands/completion/fish",
-		"/commands/completion/zsh",
-		"/dag",
-		"/dag/get",
-		"/dag/resolve",
-		"/dag/stat",
-		"/dag/export",
-		"/get",
-		"/ls",
-		"/name",
-		"/name/resolve",
-		"/object",
-		"/object/data",
-		"/object/get",
-		"/object/links",
-		"/object/stat",
-		"/refs",
-		"/resolve",
```

Comment:
:thought_balloon: if someone needs replacement for `/api/v0/resolve` we could tell them to use `x-ipfs-roots`:

```console
$ curl https://ipfs.io/ipns/en.wikipedia-on-ipfs.org/favicon.ico -X HEAD -is | grep -i ipfs-roots
x-ipfs-roots: bafybeiaysi4s6lnjev27ln5icwm6tueaw2vdykrtjkwiphwekaywqhcjze,bafkreicl5473u3g5c2l34n5oh5trtpft56ep6mbqv57affzgkfppcitfl4
```